modules/EvseManager/EvseManager.cpp

```diff
@@ -29,6 +29,22 @@ void EvseManager::init() {
     if (!(config.ac_hlc_enabled || config.ac_with_soc || config.charge_mode == "DC")) {
         slac_enabled = false;
     }
+
+    // Use SLAC MAC address for Autocharge if configured.
+    if (config.autocharge_use_slac_instead_of_hlc && slac_enabled) {
+        r_slac[0]->subscribe_ev_mac_address([this](const std::string& _token) {
```

Comment:
not sure, here, but you could change the signature into std::string token, than you'll capture directly a copy